thundra/plugins/trace/trace_support.py

```diff
@@ -58,36 +54,33 @@ def _get_sl_class(sl_class_name):
     return sl_class
 
 
-def _get_class_and_config_parts(env_v):
-    m = p1.match(env_v)
-    if m is not None:
-        try:
-            sl_class_name = m.group(1)
-            config_str = m.group(2)
-            return (sl_class_name, config_str)
-        except IndexError as e:
-            logger.error(("couldn't parse thundra span "
-                          "listener environment variable: %s"), e)
-
-    return (None, None)
-
-
 def parse_span_listeners():
     # Clear before parsing to prevent appending duplicate span listeners
     clear_span_listeners()
     # Add span listeners configured using environment variables
     for env_k, env_v in os.environ.items():
         if env_k.startswith(constants.THUNDRA_LAMBDA_SPAN_LISTENER):
```

Comment:
`THUNDRA_LAMBDA_SPAN_LISTENER` constant is now `thundra.agent.trace.span.listenerConfig`. We changed it because new format is different and old agent versions should not detect it with the old config name

Reply:
In old agents, the span listener config is thundra.agent.trace.span.listener and is checked with startswith. So new param "thundra.agent.trace.span.listenerConfig" will also pass that check in old agents. Should we use a different name ?